wire-ios-sync-engine/Tests/Source/Synchronization/Strategies/TeamInvitationRequestStrategyTests.swift

```diff
@@ -142,6 +146,10 @@ class TeamInvitationRequestStrategyTests: MessagingTest {
         }
     }
 
+    private func sutNextRequest(for apiVersion: APIVersion) -> ZMTransportRequest? {
+       return syncMOC.performAndWait { sut.nextRequest(for: apiVersion) }
```

Comment:
```suggestion
       syncMOC.performAndWait { sut.nextRequest(for: apiVersion) }
```
Maybe this is rather personal preference.. don't know if we have a guideline for `return`..